src/service/video-manager-impl.js

```diff
@@ -692,83 +692,84 @@ class VideoEntry {
     });
   }
 
   /**
-   * Called by autoplayVideoBuilt_ when an interactive autoplay video is built.
-   * It handles hiding controls, installing autoplay animation and handling
-   * user interaction by unmuting and showing controls.
+   * Installs autoplay animation and interaction mask when interactive.
    * @private
    */
-  autoplayInteractiveVideoBuilt_() {
+  installAutoplayArtifacts_() {
     const {video} = this;
+    const {element} = this.video;
 
-    // Hide the controls.
-    video.hideControls();
+    if (element.hasAttribute(VideoAttributes.NO_AUDIO) ||
+        element.signals().get(VideoServiceSignals.USER_INTERACTED)) {
+      return;
+    }
 
-    // Create autoplay animation and the mask to detect user interaction.
     const animation = this.createAutoplayAnimation_();
-    const mask = this.createAutoplayMask_();
+
+    /** @param {boolean} isPlaying */
+    const toggleAnimation = isPlaying => {
+      video.mutateElement(() => {
+        animation.classList.toggle('amp-video-eq-play', isPlaying);
+      });
+    };
+
     video.mutateElement(() => {
-      const {element} = this.video;
       element.appendChild(animation);
-      element.appendChild(mask);
     });
 
-    // Listen to pause, play and user interaction events.
-    const {element} = video;
     const unlisteners = [
-      listen(mask, 'click', triggerUserInteracted.bind(this)),
-      listen(animation, 'click', triggerUserInteracted.bind(this)),
       listen(element, VideoEvents.PAUSE, () => toggleAnimation(false)),
       listen(element, VideoEvents.PLAYING, () => toggleAnimation(true)),
-      listen(element, VideoEvents.AD_START, adStart.bind(this)),
-      listen(element, VideoEvents.AD_END, adEnd.bind(this)),
     ];
 
-    video.signals().whenSignal(VideoServiceSignals.USER_INTERACTED)
-        .then(onInteraction.bind(this));
-
-    /**
-     * @param {boolean} isPlaying
-     */
-    function toggleAnimation(isPlaying) {
-      video.mutateElement(() => {
-        animation.classList.toggle('amp-video-eq-play', isPlaying);
-      });
-    }
-
-    function triggerUserInteracted() {
-      userInteractedWith(video);
-    }
-
-    function onInteraction() {
+    video.signals().whenSignal(VideoServiceSignals.USER_INTERACTED).then(() => {
       const {video} = this;
+      const {element} = video;
       this.firstPlayEventOrNoop_();
       video.showControls();
       video.unmute();
       unlisteners.forEach(unlistener => {
         unlistener();
       });
-      removeElement(animation);
-      removeElement(mask);
-    }
+      const animation = element.querySelector('i-amphtml-video-eq');
+      const mask = element.querySelector('i-amphtml-video-mask');
+      if (animation) {
+        removeElement(animation);
+      }
+      if (mask) {
+        removeElement(mask);
+      }
+    });
 
-    function adStart() {
-      setStyles(mask, {
-        'display': 'none',
-      });
+    if (!this.video.isInteractive()) {
```

Comment:
how does this end up being `true` in stories without `control` ?  My understanding is Stories wants this icon to show up but not use `controls` attribute on videos.

Reply:
This changes the semantics of autoplay somewhat, the equalizer icon will be displayed for all videos including those without `controls`. I figured by having the `noaudio` flag we're covered?

Reply:
that would be a breaking change for sites like bmw.com. The idea was if 'controls' is not specified, the `video` is essentially a background design creative and equalizer icon should not show up.

Reply:
Maybe we can explicitly expose an attribute to show the equalizer?

Reply:
@newmuis I think making a CSS exception for stories i.e.

```csss
.amp-video-eq { display: none }
[controls] .amp-video-eq,